### ISSUE TITLE
Change List display

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/COLOGNE#324]
The 'List displays'  such as https://www.sanskrit-lexicon.uni-koeln.de/scans/AP90Scan/2020/web/webtc1/index.php
use a 'Preference' system that is not used anywhere else.

![image](https://user-images.githubusercontent.com/6393033/102700859-22c9c500-421f-11eb-9141-685340c85a41.png)

**Comment by funderburkjim** [comment]
It also allows users to input Devanagari script in various was by certain Preference choices, such as:

![image](https://user-images.githubusercontent.com/6393033/102700905-910e8780-421f-11eb-8970-31755275bd6c.png)

And in this scenario,  one has access to an onscreen keyboard:

![image](https://user-images.githubusercontent.com/6393033/102700937-d468f600-421f-11eb-90dc-63e498570e81.png)

**Comment by gasyoun** [comment]
> And in this scenario, one has access to an onscreen keyboard:

Does anybody actually use it or is aware of it? I guess it's a bit year 2000.

**Comment by funderburkjim** [comment]
These features were designed by Peter Scharf and implemented  in Javascript by Ralph Bunker.

However,  there are now other ways to enter Devanagari  (and also IAST); some are discussed in #16.

And other displays allow input of Devanagari or IAST: For example:

![image](https://user-images.githubusercontent.com/6393033/102701127-bd2b0800-4221-11eb-9f6f-574c9661ae66.png)

Here, it is up to the user to type (or copy-paste) Unicode Devanagari or IAST.

**Comment by funderburkjim** [comment]
**I propose that we phase out the List display.**

While it was good for its time,  it is no longer needed.

And the Javascript involved is extremely complex and unmaintainable.

Comments?

**Comment by gasyoun** [comment]
> While it was good for its time, it is no longer needed.

Agreed. It was good in 2000. No practical need in 2020.

**Comment by drdhaval2785** [comment]
I agree on phasing out list display. If I want to see words nearby word 'kamala', I can do prefix desrch with 'kama' in Advanced Search. This would roughly give me what List Display used to give. So, not a major functionality loss by removing list display. 

I vote for closure of List Display.

**Comment by gasyoun** [comment]
> If I want to see words nearby word 'kamala', I can do prefix search with 'kama' in Advanced Search. 

No, no, no. I our you have misunderstood something. Or, @funderburkjim ? It was all about the Devanagari entry, no?

**Comment by drdhaval2785** [comment]
> I propose that we phase out the List display.

 It is about phasing out the whole list display itself, and not some part of the code. This is what I understood. And I am OK with list display going away.

**Comment by funderburkjim** [comment]
There are two possibilities for what 'phase out' would mean:
1. Keep list display, but remove the 'Preferences', little keyboard, etc.   and
    replace with simple 'input' selector, as in Basic Display
2. Remove list display entirely (the 'L' of homepage 'B L A M D' removed)

Probably I was thinking of the first option initially, because of the unmaintainability
of the Javascript module,  and the fact that there are now satisfactory ways for users to
enter Unicode Devanagari  (or other Unicode Indian language text) as well as IAST.

**Comment by gasyoun** [comment]
> I was thinking of the first option initially

Me too. Until we have a way to show all entries on the physical page, List is needed.
